brave/browser-laptop
# Adds balance breakdown

Resolves #14229

Empty wallet
![image](https://user-images.githubusercontent.com/9574457/40555387-32599c3a-6049-11e8-8723-34f53329ec59.png)


With user fund
![image](https://user-images.githubusercontent.com/9574457/40555445-731ab0ec-6049-11e8-8623-f50aa00f040c.png)



With grant
![image](https://user-images.githubusercontent.com/9574457/40555366-1e7c1b5c-6049-11e8-8844-044f01f869a5.png)


With grant + user fund
![image](https://user-images.githubusercontent.com/9574457/40555451-7d0eb8dc-6049-11e8-8196-f475fc51515a.png)

Contribution in progress
![image](https://user-images.githubusercontent.com/9574457/40556628-a216698c-604d-11e8-91d0-b454ff637d49.png)

Contribution done
![image](https://user-images.githubusercontent.com/9574457/40557032-113f4756-604f-11e8-8876-7b84a7410a70.png)



## Submitter Checklist:


## Test Plan:
- clean profile
 - enable wallet
- add funds
- claim promotion
- do a contribution
Make sure that in all this steps you are getting correct amounts listed in the account balance section

## Reviewer Checklist:


Tests
